integration-tests/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "integration_tests"
+version = "0.0.0"
+authors = ["Exonum Team <exonum@bitfury.com>"]
+publish = false
+
+[dependencies]
+exonum = "0.6.0"
```

Comment:
Do we need to use 0.7 core version here?

Reply:
Right now, the actual used version is a manually specified commit (see the `[patch.crates-io]` section of the file). After `v0.7` is published, this can be changed to `exonum = "0.7.0"` without a `[patch.crates-io]` override.